mifosng-android/src/main/java/com/mifos/mifosxdroid/online/DashboardActivity.java

```diff
@@ -93,6 +93,13 @@ public void onDrawerOpened(View drawerView) {
                 super.onDrawerOpened(drawerView);
                 setUserStatus(userStatusToggle);
             }
+
+            @Override
+             public void onDrawerSlide(View drawerView, float slideOffset) {
+                if (slideOffset != 0 )
+                    hideKeyboard(mDrawerLayout);
+                super.onDrawerSlide(drawerView, slideOffset);
```

Comment:
I'm sorry I didn't get you. Do you mean to enclose the `if` block within braces? I'll surely keep that in mind. However, please let me know if you meant something else.